Army/ArmyBloodMoonToken.cs

```diff
@@ -0,0 +1,64 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreAdvanced.cs
+//cs_include Scripts/Army/CoreArmyLite.cs
+using Skua.Core.Interfaces;
+using Skua.Core.Options;
+
+public class ArmyBloodMoonToken
+{
+    private IScriptInterface Bot => IScriptInterface.Instance;
+    private CoreBots Core => CoreBots.Instance;
+    private CoreFarms Farm = new();
+    private CoreAdvanced Adv => new();
+    private CoreArmyLite Army = new();
+
+    private static CoreArmyLite sArmy = new();
+
+    public string OptionsStorage = "ArmyBloodMoonToken";
+    public bool DontPreconfigure = true;
+    public List<IOption> Options = new List<IOption>()
+    {
+        sArmy.player1,
+        sArmy.player2,
+        sArmy.player3,
+        sArmy.player4,
+        sArmy.player5,
+        sArmy.player6,
+        sArmy.player7,
+        sArmy.packetDelay,
+        CoreBots.Instance.SkipOptions
+    };
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.BankingBlackList.AddRange(Loot);
+
+        Core.SetOptions(disableClassSwap: true);
+        bot.Options.RestPackets = false;

```

Comment:
Beauty thing, but Bot with capital b

Reply:
For this, I just find out if all army scripts are using b instead B xD